airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ReplicationView.tsx

```diff
@@ -126,10 +118,6 @@ export const ReplicationView: React.FC<ReplicationViewProps> = ({ onAfterSaveSch
       ...connectionAsUpdate,
       ...values,
       connectionId,
-      // Use the name and status from the initial connection because
```

Comment:
i'm unclear whether removing these makes a functional difference.  i'd test that this still behaves as expected.

Reply:
I think this may deserve a ConnectionContext hoisted way above, that interfaces with the ConnectionName component as well.

Reply:
Agree.  Part of the benefit of moving the form and context around is so `ConnectionFormContext` can wrap all components changing the connection in any way (name, connectionform, refreshcatalog, operations, etc.)